docs/decisions/0004-managing-django-settings.rst

```diff
@@ -0,0 +1,97 @@
+Status
+======
+
+Accepted
+
+
+Context
+=======
+
+Settings are confusing right now and there are way too many layers. We have
```

Comment:
Nit: we [had decided to use auto-wrap](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#best-practices) rather than having fixed line lengths in our doc files.

Reply:
I'll fix this before merging, I wrapping for review because it makes the diffs a bit more readable as I make changes.  Also allows for better pinpointing of where to make the change when that's relevant.